var/spack/repos/builtin/packages/pism/package.py

```diff
@@ -76,7 +77,8 @@ class Pism(CMakePackage):
     depends_on('netcdf-c')    # Only the C interface is used, no netcdf-cxx4
     depends_on('petsc')
     depends_on('udunits')
-    depends_on('proj@:4')
+    depends_on('proj@4:', when='@:1.1')
```

Comment:
Should this be:
```suggestion
    depends_on('proj@:4', when='@:1.1')
```
?

Reply:
yes, that is what I meant.